src/dal_queryset_sequence/views.py

```diff
@@ -31,10 +31,13 @@ def has_more(self, context):
 
     def mixup_querysets(self, qs):
         """Return a queryset with different model types."""
-        if len(list(qs.query._querysets)):
-            limit = int(self.paginate_by / len(qs.query._querysets))
-            qs.query._querysets[0][:2]
-            qs = QuerySetSequence(*[q[:limit] for q in qs.query._querysets])
+        querysets = (qs.query._querysets  # querysetsequence < 0.9
+                     if hasattr(qs, 'query')
+                     else qs._querysets)  # querysetsequence >= 0.9
```

Comment:
Could this use `try … except AttributeError` (preferring the newer case)?
Also above then.

Reply:
You need to write test for both try and except block otherwise the coverage falls, so for that I prefer terneries. What do you suggest?

Reply:
Well, if branch coverage would be used the result is the same, and it is better in general to admit that some branch is not tested, isn't it? :)

```python
try:
    querysets = qs._querysets  # querysetsequence >= 0.9
except AttributeError:
    querysets = qs.query._querysets  # querysetsequence >= 0.9
```

Reply:
We have an issue to use a public api instead but i didn't make it to explain it properly or something https://github.com/percipient/django-querysetsequence/issues/21